test/test_images/latencymako/sender_receiver.go

```diff
@@ -319,8 +318,12 @@ func parsePaceSpec() ([]paceSpec, error) {
 	return pacerSpecs, nil
 }
 
-func warmup(warmupSeconds uint) {
-	cancelCeReceiver := startCloudEventsReceiver(func(event cloudevents.Event) {})
+func (ex *senderReceiverExecutor) warmup(warmupSeconds uint) error {
```

Comment:
If `startCloudEventsReceiver` gets a ctx parameter, then `warmup` has to get one too. It can create a child context of the given context, send it to `startCloudEventsReceiver`, and defer cancel.